airgun/entities/host_new.py

```diff
@@ -975,6 +975,22 @@ def refresh_applicability(self, entity_name):
         self.browser.plugin.ensure_page_safe()
         view.dropdown.item_select('Refresh applicability')
 
+    def read_update_value(self, entity_name, key, value):
+        view = self.navigate_to(self, 'NewDetails', entity_name=entity_name)
+        wait_for(lambda: view.ansible.variables.table.is_displayed, timeout=10)
+        view.ansible.variables.table.row(name=key)[5].widget.click()
```

Comment:
Can you please add some comment to the `5` index

Reply:
Added comment for the [5] index.